packages/components/src/icon/stories/index.js

```diff
@@ -1,128 +0,0 @@
-/**
- * External dependencies
- */
-import { number } from '@storybook/addon-knobs';
-
-/**
- * WordPress dependencies
- */
-import { SVG, Path } from '@wordpress/primitives';
-import { wordpress } from '@wordpress/icons';
-
-/**
- * Internal dependencies
- */
-import Icon from '../';
-import { VStack } from '../../v-stack';
-
-export default {
-	title: 'Components/Icon',
-	component: Icon,
-	parameters: {
-		knobs: { disable: false },
-	},
-};
-
-const IconSizeLabel = ( { size } ) => (
-	<div style={ { fontSize: 12 } }>{ size }px</div>
-);
-
-export const _default = () => {
-	const size = number( 'Size', '24' );
```

Comment:
I removed this explicit 24px default, since we should just showcase the default component behavior here (which is 20 when `icon` is a Dashicon, and 24 when otherwise).